torch_xla/core/xla_env_vars.py

```diff
@@ -28,6 +28,8 @@
 TPU_MESH_CTLER_PORT = 'TPU_MESH_CONTROLLER_PORT'
 PJRT_DEVICE = 'PJRT_DEVICE'
 PJRT_SELECT_DEFAULT_DEVICE = 'PJRT_SELECT_DEFAULT_DEVICE'
+PJRT_LOCAL_PROCESS_RANK = 'PJRT_LOCAL_PROCESS_RANK'
```

Comment:
Do we need to worry about the usage of LOCAL_RANK in our cpp code?
https://github.com/pytorch/xla/blob/master/third_party/xla_client/env_vars.cc#L27

Reply:
Good catch! Yeah, I need to handle that for GPU intialization.